pkgs/applications/editors/lapce/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgs }:
```

You should pass all dependencies instead of `pkgs`
```suggestion
{ lib
, fetchFromGitHub
, rustPlatform
, cmake
, pkg-config
, python3Full
, perl
, freetype
, fontconfig
, libxkbcommon
, xcbutil
, libX11
, libXcursor
, libXrandr
, libXi
, vulkan-loader
}:
```